pkg/server/server.go

```diff
@@ -520,16 +530,24 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 			connID := pkt.GetData().ConnectID
 			data := pkt.GetData().Data
 			klog.V(5).InfoS("Received data from connection", "bytes", len(data), "connectionID", connID)
+			if backend == nil {
+				klog.V(2).InfoS("Backend has not been initialized for the connection. Client should send a Dial Request first", "connectionID", connID)
```

Comment:
Seems more likely the client sent the data packet on the wrong stream than that they attempted to send a data packet before a dial request.

Reply:
Agreed. This was copied from the log message under the `DATA` case, but it's weird in both places. I just removed the second sentence.